### PR TITLE
Copy changes

### PR DESCRIPTION
**Pull request by ciccionamente**
- Updated the last paragraph of the landing page
- Added a donation link in done.php page

### CODE REVIEW

File: htdocs/PHP/internal_libraries/languages/page_content_fr.php [hunk 1]
```diff
@@ -51,8 +51,8 @@
...
 
-$translation["landing_page_section_5_title"] = "À but non lucratif";
-$translation["landing_page_section_5_description"] = "WeExpire est un projet à but non lucratif et est entièrement gratuit. Si vous avez envie d'aider le projet, vous pouvez tout de même contribuer avec un <a href='https://www.buymeacoffee.com/ciccionamente' target='_blank'>don</a>.";
+$translation["landing_page_section_5_title"] = "Rapide et à but non lucratif";
+$translation["landing_page_section_5_description"] = "Sur WeExpire vous pouvez créer votre note d'urgence en moins de 3 minutes, gratuitement. Nous croyons que l'écriture d'un dernier message numérique doit être quelque chose de facile et abordable pour n'importe qui dans le monde. Si vous avez envie d'aider le projet, vous pouvez tout de même contribuer avec un <a href='https://www.buymeacoffee.com/ciccionamente' target='_blank'>don</a>.";
```
Comment (ciccionamente) [inline htdocs/PHP/internal_libraries/languages/page_content_fr.php:55]: Hey @eze-kiel, does this sound right in French?

File: htdocs/PHP/internal_libraries/languages/page_content_fr.php [hunk 2]
```diff
@@ -93,6 +93,7 @@
 $translation["done_page_content"] = "
 <p class='text-center'>Votre note d'urgence a été créée. Vous pouvez le télécharger en cliquant sur le bouton ci-dessous, puis l'imprimer et la remettre à votre personne de confiance.</p>
 <p class='text-center'><b><mark>Très important !</mark></b> Ajoutez l'adresse e-mail <b>support@weexpire.org</b> à la liste des expéditeurs sûrs de votre fournisseur de mails pour éviter que toute notification de notre part concernant la note que vous venez de créer ne se retrouve par erreur dans les spams.</p>";
+$translation["done_page_donate_message"] = "<p class='text-center'><small>Soutenez WeExpire avec un <a href='https://www.buymeacoffee.com/ciccionamente' target='_blank'>don</a>.</small></p>";
```
Comment (ciccionamente) [inline htdocs/PHP/internal_libraries/languages/page_content_fr.php:96]: @eze-kiel I think it's correct in French but it's always better to double check it :)

File: htdocs/PHP/internal_libraries/languages/page_content_fr.php (hunk 1 above)
Comment (eze-kiel) [reply htdocs/PHP/internal_libraries/languages/page_content_fr.php:55]: this one is perfect :ok_hand:

File: htdocs/PHP/internal_libraries/languages/page_content_fr.php (hunk 2 above)
Comment (eze-kiel) [reply htdocs/PHP/internal_libraries/languages/page_content_fr.php:96]: :+1:

**Review by eze-kiel — APPROVED** [review]
French updates look good to me! :muscle:

**Comment by ciccionamente** [comment]
Thank you so much @eze-kiel , I appreciate your help! Please let me know in case you prefer to don't be involved the next time I need to double check some copy changes in French - I can totally understand.

**Comment by eze-kiel** [comment]
No worries, it's always a pleasure to help! Do not hesitate to ping me again if needed!

**Pull request merged**